src/bluesky/run_engine.py

```diff
@@ -873,7 +864,13 @@ def _create_result(self, plan_return):
         )
         return rs
 
-    def __call__(self, *args, **metadata_kw):
+    def __call__(
+        self,
+        plan: typing.Union[MsgGenerator, typing.Iterable[Msg]],
+        subs: typing.Optional[Subscribers] = None,
+        /,
+        **metadata_kw: typing.Any,
+    ):
```

Comment:
```suggestion
    def __call__(
        self,
        plan: typing.Iterable[Msg],
        subs: typing.Optional[Subscribers] = None,
        /,
        **metadata_kw: typing.Any,
    ):
```

A Generator is an Iterator is an Iterable, but it may be clearer to read for us that it explicitly accepts a MsgGenerator (if we made use of the P that it returns it definitely would)